language-support/hs/bindings/examples/nim/package.yaml

```diff
@@ -0,0 +1,37 @@
+name:                nim
```

Comment:
I’m strongly opposed to using `package.yaml` files here. You are only adding an additional layer of complexity and make it harder for people that want to use `cabal-install` instead of `stack`.

Reply:
fwiw if we go back to building sdist tarballs I care less about this since the sdist tarballs contain the cabal files but I would still prefer to just go with cabal files.

Reply:
We have the sdist tarballs back as we discussed yesterday.
But we still use stack (again as discussed yesterday)
Everyone is happy :)